windows/helpers/phase1/install_codecov.ps1

```diff
@@ -0,0 +1,38 @@
+param (
+    [Parameter(Mandatory=$true)][string]$Version,
+    [Parameter(Mandatory=$true)][string]$Sha256
+)
+
+$CodecovPath = "c:\program files\codecov"
+$CodecovUrl = "https://uploader.codecov.io/$Version/windows/codecov.exe"
+$OutFile = "codecov.exe"
+
+if (Test-Path -Path "$CodecovPath\codecov.exe") {
+    Write-Output "$CodecovPath\codecov.exe already exists on the system."
+} else {
+    Write-Output "Downloading codecov.exe to $CodecovPath"
+    Get-RemoteFile -RemoteFile $CodecovUrl -LocalFile $OutFile -VerifyHash $Sha256
+
+    # Moving codecov.exe to "c:\program files\codecov"
+    if (-Not (Test-Path -Path $CodecovPath)) {
+        Write-Output "Creating $CodecovPath"
+        mkdir $CodecovPath
+    }
+    Write-Output "Moving codecov.exe to $CodecovPath"
+    Move-Item -Path $OutFile -Destination "$CodecovPath\codecov.exe"
+}
+
+
+# Adding codecov folder to the system PATH
+$Path = [System.Environment]::GetEnvironmentVariable("PATH", "Machine")
+
+# Checking if codecov folder is already in the path
+if ($Path -notlike "*$CodecovPath*")
```

Comment:
There is an `Add-ToPath()` helper which does much the same thing.
Will also allow you to add to the container path if this is needed later in the container creation

Reply:
Didn't know about this helper, thanks Derek ! Fixed with [c290282](https://github.com/DataDog/datadog-agent-buildimages/pull/582/commits/c2902820549f063cf442d68b5f41eef08d5ffa71).